go.mod

```diff
@@ -312,6 +313,7 @@ require (
 	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac // indirect
 	google.golang.org/genproto v0.0.0-20220107163113-42d7afdf6368 // indirect
 	google.golang.org/grpc v1.43.0 // indirect
+	gopkg.in/guregu/null.v2 v2.1.2 // indirect
```

Comment:
Oh wait, we have v4 already - should we be aligned on that?

Reply:
yeah that would be good...trying to figure out where v2 started showing up

Reply:
actually, not from solana/terra/libocr, and can't figure out where....
```
$ go mod why gopkg.in/guregu/null.v2
# gopkg.in/guregu/null.v2
(main module does not need package gopkg.in/guregu/null.v2)
```
:shrug: https://github.com/golang/go/issues/27900

Reply:
Could be indirect and _test only_? https://github.com/golang/go/issues/30206

Reply:
...yeah i'm struggling to find this too. will give it another shot tomorrow 🙃

Reply:
notes: 
* this likely happened because of bumping the core version in solana & terra repos to core 1.13 so go mod chose differently